k8s/02_storage.yaml

```diff
@@ -22,14 +22,14 @@ metadata:
   labels:
     type: local
 spec:
-  storageClassName: yona-default
-  persistentVolumeReclaimPolicy: Recycle
+  storageClassName: default
+  persistentVolumeReclaimPolicy: Delete
   capacity:
     storage: 3Gi
   accessModes:
     - ReadWriteOnce
   hostPath:
-    path: "/data/yona-persistentvolumes/pv0002"
+    path: "/tmp/.yona-persistentvolumes/pv0002"
```

Comment:
This requires a change on the hosts, right?

Reply:
This only gets used on the dev-build as best I understand.  The mobiquity does not use the 'infrastructure' components.   /tmp should work OK for the build server with space being the only issue (7G free currently, and db/ldap will not take much in the bootstrap mode).

Reply:
Alternatively, I can merge it and push it through the pipeline at a moment you're available to help resolve any issues that might occur.